pom.xml

```diff
@@ -1155,6 +1155,7 @@
         <module>repository-springboot</module>
         <module>repository-wildfly</module>
         <module>karaf-itests</module>
+        <module>jbpm-workitem-itests</module>
```

Comment:
I believe this should be also added to the productized profile. Otherwise we won't be available to run these tests as part of QE certifications. @mareknovotny am I right?

Reply:
i am not sure how you run tests for binaries ;)

Reply:
added, thanks a lot for your review @MarianMacik

Reply:
I think if it's productized it will affect also to the total number of modules, than @jstastny-cz modified also here:
https://github.com/kiegroup/jbpm-work-items/pull/156
So, when that PR is merged, I'd increase also here

Reply:
@mareknovotny I rather meant that these are the only modules which are built by productization. So if we don't include it there (in the productized profile) we won't see this module and its tests in the product -> no testing.
@gmunozfe thanks for increase as well